apps/ai-image-tagging/lambda/usage.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { promisify } = require('util');
+
+const AWS = require('aws-sdk');
+
+module.exports = async spaceId => {
+  const ddb = new AWS.DynamoDB();
+  const updateItem = promisify(ddb.updateItem.bind(ddb));
+
+  const now = new Date();
+  const year = `${now.getFullYear()}`;
+  const month = `${now.getMonth()+1}`;
+  const formattedMonth = month.length === 1 ? `0${month}` : month;
+  const period = [year, formattedMonth].join('-');
```

Comment:
The standard approach for timeseries is to express the timestamp as number of ms since epoch truncated to the granularity